examples/src/main/java/org/bitcoinj/examples/ForwardingService.java

```diff
@@ -103,6 +107,8 @@ public ForwardingService(Address forwardingAddress, BitcoinNetwork network) {
                 KeyChainGroupStructure.BIP32,
                 new File("."),
                 getPrefix(network));
+        if (peerAddress != null)
+            kit.setPeerNodes(peerAddress);
```

Comment:
Actually, I don't think we're allowed to do this here. We have to wait for setup completed, I think.